gradle.properties

```diff
@@ -12,7 +12,7 @@ POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/ThePalaceProject/android-htt
 POM_SCM_URL=http://github.com/ThePalaceProject/android-http
 POM_URL=http://github.com/ThePalaceProject/android-http
 VERSION_NAME=1.2.2-SNAPSHOT
-VERSION_PREVIOUS=1.2.1
+VERSION_PREVIOUS=1.1.0
 
```

Comment:
This is out of date now. Sorry!

Reply:
What do you mean? I've changed this because of that minor/patch error on the pipeline. Has the current version changed?